plugins/linear/server/api/linear.ts

```diff
@@ -103,6 +106,13 @@ router.get(
       { transaction }
     );
 
+    transaction.afterCommit(async () => {
```

Comment:
We should probably generalize this soon, perhaps some `.scheduleWithCtx` or equivalent